### PR TITLE
Remove auto-scrolling behavior

### LINKED ISSUE #92: Remove auto-scrolling behavior when loading detail page
This seemed like it would be useful — but it seems to be error prone and would be better to just load as per normal.

### PR DESCRIPTION
**Pull request by reefdog**
This doesn’t work reliably, and even when it does, feels stranger than I expected it would. It will also be less necessary once we aren’t showing the welcome text on every page.

Resolves #92

### CODE REVIEW

**Comment by reefdog** [comment]
Because of the requirement that we check in build assets, and how that means hell for merge conflicts with simultaneous PRs, I'm going to put this into draft status until #90 is merged.

**Comment by hminsky2002** [comment]
@reefdog I suppose this is a bit wonky. Happy to ditch it in lieu of hiding welcome text. One minor code change request then should be g2g!

**Comment by reefdog** [comment]
> One minor code change request then should be g2g!

Ack, I'm not seeing any inline code change requests — did it not get sent?

File: front-end/src/App.vue [hunk 1]
```diff
@@ -72,17 +71,6 @@ const resourcesPrefetch = ref<Array<Resource> | undefined>(undefined);
 
 const searchListContainer = ref<HTMLDivElement | null>(null);
```
Comment (hminsky2002) [inline front-end/src/App.vue:72]: If we're removing the ref we should remove the searchListContainer variable and it's corresponding ref binding on the search-list-container div (Line 25!)

**Review by hminsky2002 — CHANGES_REQUESTED** [review]

**Comment by hminsky2002** [comment]
> > One minor code change request then should be g2g!
> 
> Ack, I'm not seeing any inline code change requests — did it not get sent?

Airplane wifi :/

**Review by hminsky2002 — APPROVED** [review]
Splendiferous, thank you @reefdog

**Pull request merged**